ui/admin/app/templates/application.hbs

```diff
@@ -70,42 +70,76 @@
                 </dd.Interactive>
               </Hds::Dropdown>
             {{/if}}
-
-            <Hds::Dropdown
-              @enableCollisionDetection={{true}}
-              data-test-side-nav-user-dropdown
-              as |dd|
-            >
-              {{#if this.session.data.authenticated.username}}
+            {{#if this.session.data.authenticated.username}}
+              <Hds::Dropdown
+                @enableCollisionDetection={{true}}
+                class='user-menu-dropdown'
+                data-test-side-nav-user-dropdown
+                as |dd|
+              >
                 <dd.ToggleButton
                   @icon='user'
                   @text={{this.session.data.authenticated.username}}
+                  {{hds-tooltip this.session.data.authenticated.username}}
+                  class='user-menu-dropdown__toggle-button'
+                />
+                <dd.Title @text={{t 'titles.authenticated'}} />
```

Comment:
Is it redundant to have text that just says `Signed in`? Is that not obvious?

Reply:
good point!